packages/deprecated-component-library-helpers/styles/color.scss

```diff
@@ -30,25 +30,25 @@ $ca-palette-peach: #f3786d;
 $ca-palette-yuzu: #ffce1e;
```

Comment:
`packages/deprecated-component-library-helpers/styles/color.scss` had drifted a little out of sync with  `component-library/styles/color.scss`. This fixes that (before `component-library/styles/color.scss` is deleted)